File: FoodStuffs.Web/ClientApp/src/models/recipe.js
export default class {
  constructor() {
    this.id = 0;
    this.categories = [];
    this.cookTimeMinutes = null;
    this.directions = "";
    this.ingredients = "";
    this.name = "";
    this.prepTimeMinutes = null;
    this.createdBy = "";
    this.createdOn = new Date();
    this.modifiedBy = "";
    this.modifiedOn = new Date();
  }
}

File: Models/Recipe.cs
using System;
using System.Collections.Generic;

namespace FoodStuffs.Data
{
    public partial class Recipe
    {
        public Recipe()
        {
            CategoryRecipe = new HashSet<CategoryRecipe>();
        }

        public int Id { get; set; }
        public string Name { get; set; }
        public string Directions { get; set; }
        public string Ingredients { get; set; }
        public int? PrepTimeMinutes { get; set; }
        public int? CookTimeMinutes { get; set; }
        public int CreatedByUserId { get; set; }
        public int ModifiedByUserId { get; set; }
        public DateTime CreatedOn { get; set; }
        public DateTime ModifiedOn { get; set; }

        public User CreatedByUser { get; set; }
        public User ModifiedByUser { get; set; }
        public ICollection<CategoryRecipe> CategoryRecipe { get; set; }
    }
}
